x-pack/plugins/lens/public/pie_visualization/suggestions.ts

```diff
@@ -33,14 +33,13 @@ export function suggestions({
 
   const [groups, metrics] = partition(table.columns, (col) => col.operation.isBucketed);
 
-  if (
-    groups.length === 0 ||
-    metrics.length !== 1 ||
-    groups.length > Math.max(MAX_PIE_BUCKETS, MAX_TREEMAP_BUCKETS)
-  ) {
+  if (metrics.length > 1 || groups.length > Math.max(MAX_PIE_BUCKETS, MAX_TREEMAP_BUCKETS)) {
     return [];
   }
 
+  const incompleteConfiguration = metrics.length === 0 || groups.length === 0;
```

Comment:
I think we need to reject these unless we are using the chart switcher. It will cause some unexpected behavior for the drag+drop suggestions or the suggestions from the current state if we allow incomplete suggestions in those cases.